src/hooks/useSuggestions.js

```diff
@@ -0,0 +1,98 @@
+import { useEffect, useMemo, useState } from 'react';
+import { collection, getDocs, query, orderBy, where } from 'firebase/firestore';
+
+import { SUGGESTIONS_STATUS } from 'constants/types';
+import { SUGGESTIONS } from 'firebase/collections';
+import firebaseData from 'firebase/firebase';
+import useAuth from 'hooks/useAuth';
+
+const { firebaseDb } = firebaseData;
+
+const useSuggestions = () => {
+  const DEFAULT_STATUS = useMemo(
+    () => SUGGESTIONS_STATUS.filter(({ isDefault }) => isDefault)[0],
```

Comment:
I think you could also use `find()` here and avoid needing to access the index
```suggestion
    () => SUGGESTIONS_STATUS.find(({ isDefault }) => isDefault),
```

Reply:
great idea, thanks!